docs/source/bft_configuration.md

```diff
@@ -0,0 +1,221 @@
+# Configuring and operating a Smart BFT ordering service
+
+**Audience**: *Smart BFT ordering node admins*
```

Comment:
change to BFT. SmartBFT is only an implementation.

Reply:
But Raft is also an implementation for example
And there are Smart BFT specific config here such as in the channel configuration section